File: GRLTest/Modeltest7/Modeltest7.c
#include <stdio.h>

double Modeltest7( double D,double B){
	double expr = 0.5*fmax(0, fmin(100.0, 0.25*B)) + 0.5*fmax(0, fmin(100.0, 0.25*D + 0.25*fmax(0, fmin(100.0, 0.25*B))));
	return expr;
}

File: GRLTest/Modeltest7/Modeltest7.js
function Modeltest7( D,B){
	expr = 0.5*Math.max(0, Math.min(100.0, 0.25*B)) + 0.5*Math.max(0, Math.min(100.0, 0.25*D + 0.25*Math.max(0, Math.min(100.0, 0.25*B))));
	return expr;
}
